docs/cloud-native-security/vuln-management-findings.asciidoc

```diff
@@ -3,18 +3,23 @@
 
 The vulnerabilities findings page displays the vulnerabilities detected by the <<vuln-management-overview, CNVM integration>>. CNVM findings include metadata such as the CVE identifier, CVSS score, severity, affected package, and fix version if available, as well as information about impacted systems.
 
-To help you prioritize remediation efforts, you can filter and sort your findings based on these fields. Clicking on a finding provides a detailed description of the vulnerability, and any available remediation information.
+To help you prioritize remediation efforts, you can filter and sort your findings based on these fields.
+
+Clicking on a finding provides a detailed description of the vulnerability, and any available remediation information.
 
 
 image::images/cnvm-findings-page.png[The Vulnerabilities tab of the Findings page]
 
 
 [discrete]
 [[vuln-findings-grouping]]
-== Group and filter findings
+== Group, sort, and filter findings
+
+You can group your data by resource by selecting `Resource` from the *Group by* menu. When data is grouped by resource, you can click on the name of a virtual machine to view all vulnerabilities that were found on it.
 
+When *Group by* is set to `None`, you can sort the Findings table by multiple fields. To do so either click the column headings, or click the *Sort fields* button to the upper left of the table. When sorting is active, the *Sort fields* button changes to *N fields sorted* (where `N` is the number of fields sorting your data), and can be used to modify or clear sorting.
```

Comment:
I've seen this line in the [Writing guidelines](https://docs.elastic.dev/content-design/tech-docs/writing-about-ui): 

> Use _x_ (italicized) for variable text in UI component names, such as: "Click Select all x rules, and then select Activate."

Does this apply here, i.e. should it be "...the **Sort fields** button changes to **_x_ fields sorted**..."?